config/environments/test.rb

```diff
@@ -72,8 +72,6 @@
   # Annotate rendered view with file names.
   # config.action_view.annotate_rendered_view_with_filenames = true
 
-  # Raise error when a before_action's only/except options reference missing actions
+  # Raise error when a before_action's only/except options reference missing actions.
   config.action_controller.raise_on_missing_callback_actions = true
-
-  config.x.read_only_database_url = "postgres://localhost/#{ENV.fetch('DB_DATABASE', 'manage_courses_backend_test')}"
```

Comment:
I think this was a work around from a long time ago but happy to put it back in and we can remove it in a more targeted PR. The only reason I removed it here is because the auto-generator did it.

https://github.com/DFE-Digital/publish-teacher-training/pull/4445/commits/4c12fceeeff54904dad3ac4c75050f527c9c0399